main.go

```diff
@@ -33,16 +33,17 @@ type Updates struct {
 }
 
 type ConfigInfo map[string]string
+var Zones string
 
 func main() {
 	port := flag.Int("p", 0, "On which port the server starting to listen")
 	privKey := flag.String("pk", " ", "Private Key Certificate Chain")
 	pubKey := flag.String("crt", " ", "Certificate Chain")
 	caKey := flag.String("ca", " ", "CA")
-	zoneDir := flag.String("z", " ", "Zone name you want to sync to")
 	files   := flag.Int("f", 0, "Amount of files to be synced to the specified zone")
-
+	flag.StringVar(&Zones, "z", " ", "Zone name you want to sync to.")
 	flag.Parse()
+	zoneDir := flag.Args()
```

Comment:
Why moving zoneDir from -z?

Reply:
To be able to sync configs to multiple zones as well.